src/rewriter.coffee

```diff
@@ -267,7 +267,8 @@ exports.Rewriter = class Rewriter
       # Added support for spread dots on the left side: f ...a
       if (tag in IMPLICIT_FUNC and token.spaced or
           tag is '?' and i > 0 and not tokens[i - 1].spaced) and
-         (nextTag in IMPLICIT_CALL or nextTag is '...' or
+         (nextTag in IMPLICIT_CALL or 
+         (nextTag is '...' and @tag(i + 2) in IMPLICIT_CALL and not @findTagsBackwards(i, ['INDEX_START', '['])) or
```

Comment:
I was thinking the same, but I'm not sure what would be a better option: a) put all tests for spread dots in one file, or b) add test cases in files (e.g. `arrays`, `assignment`, `functions`,...).

Basically, we need to cover all cases with spread dots preceding by space:
```coffeescript
{ a: 1, b ... }

{
  a:1
  b ...
}

f = (a ...) ->

[
  a
  b ...
]
```

I would opt for option (a).